### PR TITLE
Adds some fields

### PR DESCRIPTION
**Pull request by dextercd**
It's nice to have a place to document these values and collaborate. :+1:

### CODE REVIEW

**Comment by dextercd** [comment]
Please note that I decided to add two fields **out of order** in the profile page. See these commits 7177a81 8ec7976

File: save-format.md [hunk 1]
```diff
@@ -561,11 +557,163 @@ eb: byte
...
+14b2: int - 4-4
+
+18a2: int - 5-1
+
+1c9e: int - 6-1
+1ca2: int - 6-2
+1ca6: int - 6-3
+1caa: int - 6-4
+
+20aa: int - 7-5
```
Comment (gmjosack) [inline save-format.md:604]: are 7-[1234] unaccounted for?

File: save-format.md [hunk 2]
```diff
@@ -561,11 +557,163 @@ eb: byte
...
+
+## Area - Death Count
+
+```
+8b2: int - 1-1
+8b6: int - 1-2
+8ba: int - 1-3
+8be: int - 1-4
+
+cae: int - 2-1
```
Comment (gmjosack) [inline save-format.md:581]: Interesting. Does it not have separate counters for jungle vs volcana or temple vs tide pool?

File: save-format.md [hunk 3]
```diff
@@ -561,11 +557,163 @@ eb: byte
 49a: int  - Hard Wins
 49e: int  - Special Wins
 
+4a2: int  - Sum of Score
```
Comment (gmjosack) [inline save-format.md]: This is 32 bit? seems possible to cap out if so :X

File: save-format.md [hunk 4]
```diff
@@ -599,6 +747,15 @@ Tracks whether you ever viewed the player profile. Used to determine if it shoul
 290e: time - Time
 ```
 
+## Player - Character Selection
+
+```
+2a7a: byte - Player 1
```
Comment (gmjosack) [inline save-format.md:759]: Is this 0x00 for ana through 0x13 for classic guy?

**Review by gmjosack — COMMENTED** [review]
Awesome thanks for contributing!

File: save-format.md (hunk 1 above)
Comment (dextercd) [reply save-format.md:604]: Welll.. I've never gotten to 7-[1234] so I wasn't sure, but I just tested whether or not they are right before this location.. and they are. So i'll add them in another commit.

File: save-format.md (hunk 2 above)
Comment (dextercd) [reply save-format.md:581]: No, I think these are only here for the 'deadliest level' stat in the player profile. That page doesn't make a distinction so it simply doesn't store it.

File: save-format.md (hunk 3 above)
Comment (dextercd) [reply save-format.md]: I'll take a look at what happens when I set this to a very high value. but 2^31-1 is still pretty large

File: save-format.md (hunk 4 above)
Comment (dextercd) [reply save-format.md:759]: Yes

File: save-format.md (hunk 2 above)
Comment (dextercd) [reply save-format.md:579]: Good catch, its a signed 64 bit number, I'll update this field and also add the type to the "Types" section

File: save-format.md (hunk 3 above)
Comment (dextercd) [reply save-format.md]: Good catch, its a signed 64 bit number, I'll update this field and also add the type to the "Types" section

**Comment by dextercd** [comment]
@gmjosack Added two commits

File: save-format.md [hunk 5]
```diff
@@ -29,8 +25,10 @@ https://mossranking.com/discord
  * byte  - 1 Byte. Unknown type.
  * bool  - 1 Byte. 0x0 or 0x1
  * int   - 32 bit Integer. Little Endian
+ * int64 - 64 bit Integer. Little Endian
  * float - 32 bit Float. Little Endian
  * time  - 32 bit Integer. Multiply by 16.67 to get milliseconds.
+ * time  - 64 bit Integer. Multiply by 16.67 to get milliseconds.
```
Comment (gmjosack) [inline save-format.md:31]: i think this was meant to be `time64`

**Pull request merged**